string firstname
string lastname
uint8 age
